xwiki-pro-macros-api/src/main/resources/templates/viewfile/viewFileTemplate.vm

```diff
@@ -147,12 +147,20 @@
     #end
     {{/html}}
   #else
-    #continueExecutingMacro##
+    #set ($attachmentRef = $services.model.resolveAttachment($unescapedFilename))
+    #if ($xwiki.checkAccess($attachmentRef, 'view'))
```

Comment:
Was the attachment content displayed even without view rights? Or this is just for having a warning displayed?

Reply:
As discussed, this should be deleted